src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Need header comments here. This interface forms a contract between your code and client code. If there are no comments, developers implementing the client code would not know what to expect from this interface.